src/js/config/config.js

```diff
@@ -1,7 +1,7 @@
 var config = {
   // @@ENV gets replaced by build system
   environment: "@@ENV",
-  rootUrl: "",
+  rootUrl: "assets/",
```

Comment:
Should we leave this empty, because it's overridden on line 16 for production. Or other way around. Make it empty in 
```
if (process.env.GULP_ENV === "development") {
```
?